src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -589,37 +590,31 @@ export class ExposureNotificationService {
     }
 
     const {keysFileUrls, lastCheckedPeriod} = await this.getKeysFileUrls();
-    captureMessage('keysFileUrls', keysFileUrls);
+
     try {
       let exposureWindows: ExposureWindow[];
       if (Platform.OS === 'android') {
         exposureWindows = await this.exposureNotification.getExposureWindowsAndroid(keysFileUrls);
       } else {
-        const summaries = await this.exposureNotification.detectExposure(exposureConfiguration, keysFileUrls);
```

Comment:
Do we need a version check here?

Reply:
That happens higher up on line 359 inside the `doesPlatformSupportV2` fn. If the os/version doesn't support V2, then the v1 function gets called instead.